apps/hobbies-helsinki/jest.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
```

Comment:
Should use same configs as on other example apps

Reply:
not exactly the same, but close enough. Works in CI. Deployed #17